packages/katello/nodejs-downshift/nodejs-downshift.spec

```diff
@@ -4,18 +4,24 @@
 %global npm_name downshift
 
 Name: %{?scl_prefix}nodejs-downshift
-Version: 1.31.15
-Release: 2%{?dist}
+Version: 5.4.7
+Release: 1%{?dist}
 Summary: A set of primitives to build simple, flexible, WAI-ARIA compliant React autocomplete components
 License: MIT
 Group: Development/Libraries
-URL: https://github.com/paypal/downshift#readme
-Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
-%if 0%{?scl:1}
-BuildRequires: %{?scl_prefix_nodejs}npm
-%else
```

Comment:
This part was needed. I think you may have an old npm2rpm version. It's also needed in the other package.